code-style/git/.gitmessage

```diff
@@ -5,6 +5,8 @@
 #
 # <footer>
 #
+# ** A good message should complete this sentence: when applied, this commit will... **
```

Comment:
Should you replace the word "message" with "subject" (like you did in the README) to help clarify the recommendation is not for commit body?

From a similar article:
> Remember: Use of the imperative is important only in the subject line. You can relax this restriction when you’re writing the body.

https://chris.beams.io/posts/git-commit/

Reply:
Yes, good catch!

Reply:
We do mention in the README here that the body should use an imperative tone https://github.com/sparkbox/standard/blob/ab244d2331a527eb12dbd05febd30ddb1b0b1cfc/code-style/git/README.md#the-body